content/blog/2021-03-10_c2b.Rmd

```diff
@@ -0,0 +1,157 @@
+
+The ACHD team spent hours each day wrangling data and then only had time to look at a handful of the data slices worth monitoring. The ACHD was worried that they would miss a rise in cases in specific demographics due to this confluence of factors. 
+
+As the Delphi Group tried to develop solutions to these problems, we consistently ran into data access challenges and bureaucratic limitations to develop solutions at the granularity needed. Carnegie Mellon University did not have a Business Associates Agreement, or BAA, in place with the ACHD. Due to HIPAA laws, that meant we could only operate on public, aggregated and/or redacted data sets. BAAs are commonplace in healthcare, but it is a complex legal agreement that can take months to execute, even in normal circumstances. Without a BAA in place, we couldn't use detailed COVID testing data classified as Protected Health Information (PHI). Intended projects like setting up mobile testing sites in specific neighborhoods became impossible because public data is aggregated only to the county level. We could not do finer-grained intelligence.
+
+Cases2Beds<sup>[1](#ACHDDashboardLink)</sup> was more feasible than these monitoring projects because we didn't need full line-level case and test data, and we could provide intelligence at a county level. 
+
+## A probabilistic approach
+
+To extrapolate beds utilization 1-2 weeks in the future, we needed to estimate:
+
+1. The probability that a person who tested positive for COVID-19 would require a hospitalization
+1. How many days after the test would a person be hospitalized
+1. How long a person would stay in the hospital
+1. The current number of positive COVID tests
+
+These numbers varied widely across three demographic factors: age, sex, and race.
+
+![**Fig. 2:** Demographic Comparisons from the Allegheny County COVID-19 Tableau <sup>[3](#ACHDDashboardLink)</sup>](/blog/2021-03-10_c2b/fig2.png)
+
+We used public data from Allegheny County about the number of people tested, test positivity rate, and hospitalization rate by the aforementioned demographic factors. Of these characteristics, we found that age-group was the most important.
+
+We also acquired information for two critical parameters: offset and length of stay. Offset is the number of days between the day of testing (called specimen collection date) and the first day of hospitalization. For example, if the test date is around the hospital admit date, the offset would be 0 days (or sometimes -1 or +1 days). Otherwise, if the test was 5 days before hospitalization, the offset would be 5 days. Notably, the offset can be negative, meaning you may have been tested some days or weeks after being admitted (presumably for an unrelated reason).  
+
+The length of stay is how long someone was in the hospital. The length of stay is approximately how long a person uses a bed for (± 1 day). This data helped us understand how long people would be in the hospital.
```

Comment:
Small nit, but the first two sentences read a little redundant. Maybe "The length of stay is how long someone was in the hospital; it is approximately..."

Reply:
Good catch, how about just the second sentence:


" The length of stay is approximately how many days a person uses a bed in the hospital (± 1 day). This data..."

Reply:
That sounds great to me.